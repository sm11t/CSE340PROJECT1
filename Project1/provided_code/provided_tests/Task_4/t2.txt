TASKS
    1 4
POLY
    F = 2x + 1;
EXECUTE
    W = F(4);
    INPUT W;
    OUTPUT W;
    W = F(W);
    W = F(F(W));
INPUTS
    2 4 6