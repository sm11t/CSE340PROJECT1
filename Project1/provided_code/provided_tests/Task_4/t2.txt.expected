Warning Code 2: 6 10